kolibri/plugins/learn/assets/src/views/index.vue

```diff
@@ -217,6 +219,14 @@
         if (this.pageName === PageNames.EXAM) {
           return 'exam-page';
         }
+        if (
```

Comment:
Don't really like this, but I wanted to use a component since these are all going to be so similar. I can make a separate component for each one with different headers if you all think it'd be best.